gratipay/utils/markdown.py

```diff
@@ -19,6 +19,13 @@ def render(markdown):
     ))
 
 
+def rtlo_scrub(markdown):
```

Comment:
I just experimented a bit and I see no issue. So my assumption was incorrect. I will make this into one function as @rohitpaulk suggested.